systems-at-scale.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: How to Instrument Systems at Scale
+---
+_Before getting onto recommendations on how to instrument your large-scale system with OpenTracing, be sure to read the [Specification overview](/spec)._
+
+<div id="toc"></div>
+
+
+## Spans and Relationships
+
+The two fundamental aspects of implementing OpenTracing across your infrastructure are _Spans_ and the _Relationships_ between those spans:
+
+* **_Spans_** are a logical unit of work in the system that has a start time and a duration. In a trace Spans are the components across the systems exercised on a specified path.
```

Comment:
Hi Guys! One thing to note - I noticed gRPC and some other open source projects always define the project in the beginning of documents (regardless of repetition). In our case, spans and their relationships is a key concept for the best practices doc, so it will be a good memory jog for people to read "what is a span" at the beginning. And they can go to the spec.md to get all the definitions. I agree with @yurishkuro that we should go for as much consistency as possible. I really like the idea of doing a rev on spec.md next, @azimman. Perhaps we can match the definitions then?